themes/theme-b2b/src/index.ts

```diff
@@ -46,7 +49,7 @@ const theme: Theme = {
     body: {
       margin: 0,
       padding: 0,
-      fontFamily: 'body',
+      fontFamily: "body",
```

Comment:
Do you have new prettier settings? If I do refactor this file theres no change from '' to ""

Reply:
there was no prettier in LiveShare